pytest_plugins/requirements/update_requirements.py

```diff
@@ -36,6 +36,8 @@ def pytest_report_header(config):
         if config.getoption('update_required_reqs') or config.getoption('update_all_reqs'):
             print('Updating the mandatory requirements on demand ....')
             updater.install_req_deviations()
+    else:
+        print(f'{Colored.GREEN}Mandatory Requirements are up to date!{Colored.RESET}')
```

Comment:
I feel there is no need to put that much stress on it.
```suggestion
        print(f'{Colored.GREEN}Mandatory Requirements are up to date.{Colored.RESET}')
```

Reply:
Yes, but sometimes users might think that the warning or confirmation isnt provided this plugin so may be isnt working.

Hence this is just a sign of relief.